cmd/tester/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM golang:1.9.2 as builder
+RUN mkdir -p /go/src/github.com/openfaas-incubator/connector
+WORKDIR /go/src/github.com/openfaas-incubator/connector
+
+COPY main.go    .
+
+RUN go get -d -v ./...
+
+# Stripping via -ldflags "-s -w" 
+RUN CGO_ENABLED=0 GOOS=linux go build -a -ldflags "-s -w" -installsuffix cgo -o ./connector
+
+FROM alpine
```

Comment:
Copy the example from the vcenter connector please it has an updated alpine version, we must always pin to `3.8` or whatever version we are targeting.